### ISSUE TITLE
Whatever happend to (partially) parsing XML comments in the editor tooltips?

### ISSUE THREAD

**Issue by abelbraaksma** [dotnet/fsharp#7844]
In VS 2015, the `para` tags where still parsed:

![image](https://user-images.githubusercontent.com/16015770/68696155-bf318a80-057c-11ea-993c-18b4bf595ce9.png)

In VS 2017, they were gone (but more colorful):

![image](https://user-images.githubusercontent.com/16015770/68696259-e7b98480-057c-11ea-8ba0-4e4df785a198.png)

And the didn't return in VS 2019 either:

![image](https://user-images.githubusercontent.com/16015770/68696321-0881da00-057d-11ea-8552-d619d5e86c04.png)

I'd love to have them back :100:, preferably with support for some of the XML comments features. And if I'd know where this parsing is done I might tackle this myself.

**Comment by cartermp** [comment]
XML parsing is identical on the F# side as it was in VS 2015 (modulo some minor bug fixes). @sharwell, is this a Roslyn thing/

**Comment by sharwell** [comment]
Paragraphs are separated in Quick Info for C# and VB.

**Comment by sharwell** [comment]
📝 Note that `<parameter>` is not a recognized documentation element:

1. It should be `<param name="bar">`
2. It should be located _outside_ the `<summary>` element

**Comment by abelbraaksma** [comment]
@sharwell, you're right. However, the bug persists. I see it with doc comments with only para elements, with any other allowed element.

It appears that they are parsed as concatenated text nodes, with no space in between (white space between elements is not collapsed, its removed).

**Comment by abelbraaksma** [comment]
@cartermp, I think the issue is with the F# editor after all. I tried a C# project that is referenced within the same solution as the F# project. 

Inside the C# editor, a particular comment looks like this (3 lines):

![image](https://user-images.githubusercontent.com/16015770/68946609-b5459c80-07b3-11ea-8f88-098f3a735f67.png)

And, in the same solution (after switching Generate XML Comments to "on"), this is shown in the F# editor:

![image](https://user-images.githubusercontent.com/16015770/68946667-d27a6b00-07b3-11ea-8f12-bf84f2ae5e0c.png)

I tried experimenting with `br` and `p` instead of `para`, and I vaguely remember that in the past I found a workaround, but I can't make it work now anymore, it seems.
